dev/Tickets/1443.py

```diff
@@ -83,13 +83,11 @@ def calc_isolines(fluid="HEOS::water", kind_a="P", range_a=[1e6,10e6], kind_b="T
         single_result = ResClass()
         guesses = CoolProp.CoolProp.PyGuessesStructure()
 
-        
-
         if CUR_CONF==CONF_CPP_DIR:
             set_config_bool(CoolProp.USE_GUESSES_IN_PROPSSI, False)
             start = timer()
-            try: 
-                tmp_result = PS(kind_o, kind_in1, vector_in1, kind_in2, vector_in2, fluid)	
+            try:
+                tmp_result = PS(kind_o, kind_in1, vector_in1, kind_in2, vector_in2, fluid)
             except:
                 ++err_count
```

Comment:
https://stackoverflow.com/questions/30667675/what-does-a-red-text-background-mean-in-github-comparison-view